commit 28fad1e0f04f2606e89d51d32b7bb33dce28b4c4
Author:     Zhihong Zhang <100308595+nvidianz@users.noreply.github.com>
AuthorDate: Wed Nov 26 15:43:19 2025 -0500
Commit:     GitHub <noreply@github.com>
CommitDate: Wed Nov 26 15:43:19 2025 -0500

    Moved several messages to debug level (#3869)
    
    ### Description
    
    Moved 3 messages to debug level,
    
    1. ServerEngine - WARNING - no run_manager in Server Engine.
    2. ClientRunner - INFO - [identity=site-dolly,
    run=dee67e53-1f29-4e80-8c0e-76e37e9ff8d6] - syncing to parent server ...
    3. Sent Msg Size High Water
    
    ### Types of changes
    <!--- Put an `x` in all the boxes that apply, and remove the not
    applicable items -->
    - [x] Non-breaking change (fix or new feature that would not break
    existing functionality).
    - [ ] Breaking change (fix or new feature that would cause existing
    functionality to change).
    - [ ] New tests added to cover the changes.
    - [ ] Quick tests passed locally by running `./runtest.sh`.
    - [ ] In-line docstrings updated.
    - [ ] Documentation updated.
---
 nvflare/private/fed/server/server_engine.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nvflare/private/fed/server/server_engine.py b/nvflare/private/fed/server/server_engine.py
index 0f478f0e..b2187595 100644
--- a/nvflare/private/fed/server/server_engine.py
+++ b/nvflare/private/fed/server/server_engine.py
@@ -509,7 +509,7 @@ class ServerEngine(ServerEngineInternalSpec, StreamableEngine):
             return self.run_manager.new_context()
         else:
             # this call should never be made before the run_manager is created!
-            self.logger.warning("no run_manager in Server Engine.")
+            self.logger.debug("no run_manager in Server Engine.")
             return FLContext()
 
     def add_component(self, component_id: str, component):
